crates/autopilot/src/run_loop.rs

```diff
@@ -468,8 +554,8 @@ pub struct InFlightOrders {
     orders: HashSet<domain::OrderUid>,
 }
 
-struct Participant<'a> {
-    driver: &'a infra::Driver,
+struct Participant {
+    driver: infra::Driver,
```

Comment:
I tried to avoid that, but cloning the Driver struct should not be expensive since the client field is already Arc, and the remaining fields are just strings.